src/Microsoft.Azure.WebJobs.Host/Bindings/DefaultAttributeInvokerDescriptor.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    // Default implementation of IAttributeInvokeDescriptor which can be operate on any attribute and 
+    // follows basic conventions with a [ResolveProperty]. 
+    // Assume any property without a [ResolveProperty] attribute is read-only. 
+    internal class DefaultAttributeInvokerDescriptor<TAttribute> : IAttributeInvokeDescriptor<TAttribute>
+        where TAttribute : Attribute
+    {
+        private readonly TAttribute _source;
+
+        public DefaultAttributeInvokerDescriptor(TAttribute source)
+        {
+            _source = source;
+        }
+
+        public TAttribute FromInvokeString(string invokeString)
+        {
+            if (invokeString == null)
+            {
+                throw new ArgumentNullException("invokeString");
+            }
+            AttributeCloner<TAttribute> cloner = new AttributeCloner<TAttribute>(_source);
+
+            // This has an instantiation problem similar to what Attribute Cloner has 
```

Comment:
I've updated the commented.

---
In reply to: [61139781](https://github.com/Azure/azure-webjobs-sdk/pull/694#discussion_r61139781) [](ancestors = 61139781)